static/js/main.js

```diff
@@ -1,33 +1,59 @@
 "use strict";
 
-(function()
-{
-  function initLanguageSelection()
-  {
-    var locale = document.getElementById("navbar-locale-selected");
-
-    // skip if page does not have language selection (EG: blog)
-    if (!locale) return;
-
-    locale.addEventListener("click", function()
-    {
-      document.getElementById("navbar-locale-menu")
-        .classList.toggle("visible");
-    }, false);
-  }
-
-  function navigationClick(event)
-  {
-    document.getElementById("navbar-menu")
-      .classList.toggle("visible");
-  }
-
-  function initMenu()
-  {
-    document.getElementById("navbar-menu-toggle")
-      .addEventListener("click", navigationClick, false);
-  }
-
-  initLanguageSelection();
-  initMenu();
+(function() {
+    function initLanguageSelection() {
+        var locale = document.getElementById("navbar-locale-selected");
+
+        // skip if page does not have language selection (EG: blog)
+        if (!locale) return;
+
+        locale.addEventListener("click", function() {
+            document.getElementById("navbar-locale-menu")
+                .classList.toggle("visible");
+        }, false);
+    }
+
+    function navigationClick(event) {
+        document.getElementById("navbar-menu")
+            .classList.toggle("visible");
+    }
+
+    function initMenu() {
+        document.getElementById("navbar-menu-toggle")
+            .addEventListener("click", navigationClick, false);
+    }
+
+    function navbarToggle() {
+        var navBar = document.getElementById("navbar");
+        var navbarHeight = navBar.offsetHeight;
+        var scrollHandled = false;
+        var lastScrollTop = 0;
+
+        window.addEventListener("scroll", (function() {
+            scrollHandled = true;
+        }));
+
+        setInterval(function() {
+            if(window.innerWidth>991) {
```

Comment:
NIT/Suggest: Use a constant (actually a variable in ES5; unfortunately) for 991 instead of hard coding this value to make it clearer what you're comparing to. e.g. `if (window.innerWidth > desktopBreakpoint)`